app/views/layout.erb

```diff
@@ -15,8 +15,15 @@
 <body>
   <div class="container">
     <strong>Together, we can TODO it!</strong>
+    <% if session[:user_id] %>
```

Comment:
Having a `current_user` helper would be an improvement on looking directly at the session here.